uint32 episode_id
nav_msgs/Path path
